src/patient-registration/patient-registration.test.tsx

```diff
@@ -4,6 +4,13 @@ import { render, wait, fireEvent } from '@testing-library/react';
 import * as backendController from './patient-registration.resource';
 import { PatientRegistration } from './patient-registration.component';
 
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useLocation: () => ({
+    pathname: 'openmrs/spa/patient-registration',
+  }),
+}));
+
```

Comment:
Why this is required? See: https://github.com/testing-library/react-testing-library/issues/493